docs/backend-api/resources/tracking/delivery.md

```diff
@@ -9,93 +9,136 @@ API base path: `/delivery`.
 
 ### read
 
-Returns info sufficient for tracking certain task state and the tracker assigned to it.
-Search is conducted only among tasks and checkpoints, which have start date less than or equal now and have statuses:
+Returns info sufficient for tracking certain task state, and the tracker assigned to it.
+Search conducted only among tasks and checkpoints, which have start date less than or equal now and have statuses:
 arrived, assigned or delayed.
-If multiple tasks or checkpoints were found, then return first task, otherwise checkpoint. 
+If multiple tasks or checkpoints found, then return first task, otherwise checkpoint. 
 
 #### session types:
 
-in addition to standard user session, this call supports special *DELIVERY* session type
+In addition to standard user session, this call supports special *DELIVERY* session type.
 
 #### parameters
 
 |name |description |type |format |
 |--- |--- |--- |--- |
-| external_id | an external id of task | int | 259876 |
+| external_id | An external id of task. | int | 259876 |
 
-#### example
+#### examples
 
-```abap
-$ curl -X POST '{{ extra.api_example_url }}/delivery/read' \
-  -H 'Content-Type: application/json' \ 
-  -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b", "external_id": "259876"}' 
-```
+=== "cURL"
+
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/delivery/read' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b", "external_id": "259876"}'
+    ```
+
+=== "HTTP GET"
+
+    ```
+    {{ extra.api_example_url }}/delivery/read?hash=a6aa75587e5c59c32d347da438505fc3&external_id=259876
+    ```
 
 #### response
 
 ```json
 {
     "success": true,
-    "user_id": 3, // master id of the user to which the task belongs to
-    "task" : ${task}, //a task object, for more info see task/ section
-    "tracker" : ${tracker}, //corresponding tracker object, for more info see tracker/ section
-    "restrictions": ${restrictions}, //tariff restrictions object, for more info see user/get_tariff_restrictions 
-    "first_name": "John", //first name of employee assigned to the task, or null if missing
-    "middle_name": "Micheel", //middle name of employee assigned to the task, or null if missing
-    "last_name": "Johnson", //last name of employee assigned to the task, or null if missing
-    "vehicle_label": "Service car 002", //label of the vehicle assigned to the task, or null if missing
-    "estimated_time": 1122 //estimated time of arrival in seconds, or null if unavailable
+    "user_id": 3,
+    "task" : {"task_object":},
```

Comment:
`{"task_object":}` - it's invalid example. Can you add a real example here or keep link to description?

Reply:
the link to the descriptions below. I added a real example in a new commit